modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/BaseAccountAddressResourceImpl.java

```diff
@@ -1086,6 +1093,14 @@ protected <T, R, E extends Throwable> long[] unsafeTransformToLongArray(
 	protected UnsafeBiConsumer
 		<Collection<AccountAddress>, UnsafeConsumer<AccountAddress, Exception>,
 		 Exception> contextBatchUnsafeConsumer;
+
+	protected AccountAddress updateExternalReferenceCode(
```

Comment:
the AccountAddress is not being used, so why are we defining the method like that?